xbmc/utils/SystemInfo.cpp

```diff
@@ -1243,7 +1243,8 @@ std::string CSysInfo::GetVersion()
 
 std::string CSysInfo::GetBuildDate()
 {
-  return StringUtils::Format("%s", __DATE__);
+  std::string bdate = CCompileInfo::GetBuildDate();
+  return StringUtils::Format("%s", bdate);
```

Comment:
Why not just return the string?

Reply:
dunno, the format was there before and I didn't want to mess with it

Reply:
not sure what to do with the Format here, remove?